dist/index.js

```diff
@@ -6191,12 +6191,12 @@ async function maybeInstallRebar3(rebar3Spec) {
 
 async function getOTPVersion(otpSpec0, osVersion) {
   const otpVersions = await getOTPVersions(osVersion)
-  const otpSpec = otpSpec0.match(/^(OTP-)?([^ ]+)/)
+  const otpSpec = otpSpec0.match(/^(OTP-|maint-)?([^ ]+)/)
```

Comment:
I wonder if this should be case insensitive, thoughts?

Reply:
In which case do you expect this to change? I don't mind making it insensitive to case, but it's worked thus far...